src/Control/Monad/Eff/JQuery.purs

```diff
@@ -63,6 +118,24 @@ foreign import setAttr
   -> JQuery
   -> Eff (dom :: DOM | eff) Unit
 
+foreign import getAttrImpl
+  :: forall eff
+   . String
+  -> JQuery
+  -> Eff (dom :: DOM | eff) Foreign
+
+-- | Get an attribute value.
+getAttr 
+  :: forall eff
+   . String
+   -> JQuery
+   -> Eff (dom :: DOM | eff) (Maybe String)
```

Comment:
yes, it is.
There is a bug for the setAttr, it is incorrect. Once I get some opinions, I will submit the micro fix.
https://github.com/purescript-contrib/purescript-jquery/issues/39